translations/harbour-osmscout-server-es.ts

```diff
@@ -230,47 +230,47 @@
     <message>
         <location filename="../src/routingforhuman.cpp" line="98"/>
         <source>Drive along %along%</source>
-        <translation>Continua por la %along%</translation>
+        <translation>Conducir a lo largo%a lo largo%</translation>
```

Comment:
Excellent! I don't mind to update the translation, but please then keep intact all strings between %% :)